src/processes.c

```diff
@@ -2159,8 +2173,10 @@ static int ps_read(void) {
   for (procstat_t *ps_ptr = list_head_g; ps_ptr != NULL; ps_ptr = ps_ptr->next)
     ps_submit_proc_list(ps_ptr);
 
-  read_fork_rate();
-  /* #endif KERNEL_LINUX */
+  read_fork_rate(buffer);
+  if (report_sys_ctxt_switch)
+    read_sys_ctxt_switch(buffer);
+    /* #endif KERNEL_LINUX */
 
 #elif HAVE_LIBKVM_GETPROCS && HAVE_STRUCT_KINFO_PROC_FREEBSD
```

Comment:
Does "format.sh" work better if that comment is associated with the code it refers to?
```suggestion
    read_sys_ctxt_switch(buffer);

  /* #endif KERNEL_LINUX */
#elif HAVE_LIBKVM_GETPROCS && HAVE_STRUCT_KINFO_PROC_FREEBSD
```

PS. this this is not the only place where such comments are wrongly associated & indented with the last code line, instead of the #elif/#else/#endif that actually terminates the preceeding #if/#elif/#else block.

Also, IMHO having `#endif` in the comments is misleading as it looks like commented-code instead of just a comment.  Esing e.g. "KERNEL_LINUX ends" would be clearer.

Reply:
I agree with your point, but it seems like this annotation style is prevalent in many plugins such as cpu plugin. Should we strive for consistency?

Reply:
IMHO:
* If that fixes the indenting done by `format.sh`, that would be nice, but that should be a separate PR, and some comment from current project maintainers would be nice too
* Above KERNEL_LINUX "define end" comment does not need to be changed in this PR, as it's how rests of the plugin code is written currently

Reply:
Agreed, so for now I'll leave this comment unchanged here for this PR.